src/SourceCode.Chasm/TreeId.cs

```diff
@@ -13,18 +13,6 @@ namespace SourceCode.Chasm
     [DebuggerDisplay("{" + nameof(Sha1) + ".ToString(\"D\"),nq,ac}")]
     public struct TreeId : IEquatable<TreeId>, IComparable<TreeId>
     {
-        #region Constants
-
-        /// <summary>
-        /// A singleton representing an empty <see cref="TreeId"/> value.
-        /// </summary>
-        /// <value>
-        /// The empty.
-        /// </value>
-        public static TreeId Empty { get; }
```

Comment:
Why did you get rid of this - we discussed this morning that it was functionally equivalent to `default` but that it was syntactically nicer?